docker/entrypoint

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export PATH="${HOME}/.pyenv/bin:$PATH"
```

Comment:
I am not sure if this is required, as you already have this in an `ENV` instruction.

Reply:
I'll check and remove if it is unnecessary.